chain-core/src/init/network.rs

```diff
@@ -1,5 +1,82 @@
-#[derive(Debug)]
+use std::sync::Once;
```

Comment:
I'm not sure if this "std::sync::*" is present in sgx sdk -- the TX validation wouldn't use the human-readable representation, so I think this network module could be feature-guarded in `chain-core/src/init/mod.rs`

Reply:
it would also require feature guards elsewhere in chain-core where the network's functions are used

Reply:
ok, i used bech32 for feature guard